examples/neural_networks/keras_cnn_optimization.py

```diff
@@ -16,9 +16,8 @@
 
 from examples.neural_networks.dataset import load_har
 
-from photonai.base import Hyperpipe, PipelineElement, PhotonRegistry
+from photonai import Hyperpipe, PipelineElement, PhotonRegistry, IntegerRange, BooleanSwitch
 from photonai.modelwrapper.keras_base_models import KerasBaseClassifier
```

Comment:
KerasBaseClassifier cannot be moved to __init__.py of photonai as photon would then always import keras.